language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/Template.java

```diff
@@ -3,11 +3,13 @@
 
 package com.daml.ledger.javaapi.data;
 
+import com.daml.ledger.javaapi.data.codegen.ContractId;
 import com.daml.ledger.javaapi.data.codegen.CreateAnd;
+import com.daml.ledger.javaapi.data.codegen.Update;
 
 public abstract class Template extends com.daml.ledger.javaapi.data.codegen.DamlRecord<Template> {
 
-  public abstract CreateCommand create();
+  public abstract Update<? extends ContractId<? extends Template>> create();
```

Comment:
Will change again as discussed:

```suggestion
  public abstract Update<? extends CreatedEvent<? extends ContractId<? extends Template>>> create();
```